### PR TITLE
account onboarding changes

### LINKED ISSUE #1034: Seedphrase UX improvements
Across all seed phrase: use all-caps instead of camel case. Words like "Ill" can be mistyped as 3-lowercase L's. By changing it to all caps, we remove the ambiguity. "Ill" -> "ILL"

Creating a new wallet:
* when advancing through the 12 words, require users to press "Next Word" instead of swiping. I've found swiping too aggressively can skip words, which is not good.
* instead of challenging 3 words, require users to re-enter all 12 words in the same order

Importing a wallet:
* after a user has entered all 12 words, they're asked to confirm. This confirmation used to have a carousel interaction. Let's remove that here as well and require users to press "Next Word".

Revealing Words:
* instead of swipe to go to next word, we should have a next button to advance. The swiping motion can proceed too quickly and you can skip words.

https://www.figma.com/file/x07XtWRI573FfO8duPMIEz/?node-id=1277%3A98

### LINKED ISSUE #1086: Unable to Import Account (Pixel 6 Pro Android 12)
**Describe the bug**
App does not move past entering 12-word recovery seed phrase

**To Reproduce**
Install: https://play.google.com/store/apps/details?id=com.helium.wallet
Launch app
Select "Import Existing Account"
Enter seed phrase (case insensitive)
Arrow key on keyboard does not respond
No other method to move forward  in UI

**Expected behavior**
Move to next screen after entering seed phrase

**Screenshots**
N/A (would reveal seed phrase)

**Smartphone (please complete the following information):**

- Device: Google Pixel Pro 6
- OS: Android 12
- App Version 3.8.2

**Additional context**
Existing install works on Pixel 3A XL (verified words against backup OK)

### PR DESCRIPTION
**Pull request by tyler-whitman**
- when swiping through onboarding words you can only swipe one word at a time no matter how fast you swipe
- all mnemonic words now displayed in capital letters to avoid confusion
- users must now verify all 12 words when onboarding a new account
- when horizontal scrolling the matching words list, ensure we scroll back to the start when the user types again
- dismiss the keyboard when hitting keyboard submit while entering account words

closes #1034 
closes #1086

### CODE REVIEW

File: src/features/onboarding/importAccountConfirm/ImportAccountConfirmScreen.tsx [hunk 1]
```diff
@@ -57,31 +57,39 @@ const ImportAccountConfirmScreen = () => {
...
-          }. `}</Text>
-          <Text variant="h1" color="purpleDark" maxFontSizeMultiplier={1}>
-            {upperFirst(item)}
-          </Text>
-        </Card>
-      </TouchableOpacityBox>
-    ),
-    [],
+          <Card
+            marginHorizontal="s"
```
Comment (matthewcarlreetz) [inline src/features/onboarding/importAccountConfirm/ImportAccountConfirmScreen.tsx:69]: `marginHorizontal` can probably be removed now?

File: src/features/onboarding/accountCreatePassphrase/AccountCreatePassphraseScreen.tsx [hunk 2]
```diff
@@ -41,25 +41,31 @@ const AccountCreatePassphraseScreen = () => {
...
+        marginRight={isLast ? 'l' : undefined}
+        variant="elevated"
+        flex={1}
+        overflow="hidden"
+        backgroundColor="white"
+        padding="l"
+        alignItems="center"
+        flexDirection="row"
+      >
+        <Text variant="h1" color="purpleLight" maxFontSizeMultiplier={1}>{`${
```
Comment (matthewcarlreetz) [inline src/features/onboarding/accountCreatePassphrase/AccountCreatePassphraseScreen.tsx:60]: The vertical alignment is off on these. This seems to fix it
```ts
        <Text
          variant="bold"
          fontSize={39}
          color="purpleLight"
          maxFontSizeMultiplier={1}
        >{`${index + 1}. `}</Text>
        <Text
          variant="bold"
          fontSize={39}
          color="purpleDark"
          maxFontSizeMultiplier={1}
        >
          {upperCase(item)}
        </Text>
```

File: src/features/onboarding/accountEnterPassphrase/AccountEnterPassphraseScreen.tsx [hunk 3]
```diff
@@ -126,7 +116,7 @@ const AccountEnterPassphraseScreen = () => {
           {`${index + 1}. `}
         </Text>
         <Text variant="h1" color="purpleDark" maxFontSizeMultiplier={1}>
```
Comment (matthewcarlreetz) [inline src/features/onboarding/accountEnterPassphrase/AccountEnterPassphraseScreen.tsx:118]: same vertical alignment issue here

File: src/features/onboarding/importAccountConfirm/ImportAccountConfirmScreen.tsx [hunk 4]
```diff
@@ -57,31 +57,39 @@ const ImportAccountConfirmScreen = () => {
...
+            marginRight={isLast ? 'l' : undefined}
+            variant="elevated"
+            flex={1}
+            overflow="hidden"
+            backgroundColor="white"
+            paddingHorizontal="l"
+            alignItems="center"
+            flexDirection="row"
+          >
+            <Text
```
Comment (matthewcarlreetz) [inline src/features/onboarding/importAccountConfirm/ImportAccountConfirmScreen.tsx:80]: same vertical alignment issue here as well

**Review by matthewcarlreetz — CHANGES_REQUESTED** [review]

File: src/features/onboarding/accountCreatePassphrase/AccountCreatePassphraseScreen.tsx (hunk 2 above)
Comment (tyler-whitman) [reply src/features/onboarding/accountCreatePassphrase/AccountCreatePassphraseScreen.tsx:60]: Weird bug haha, updated

**Review by matthewcarlreetz — APPROVED** [review]

**Pull request merged**